packages/client/src/Config.ts

```diff
@@ -50,64 +50,241 @@ export interface EthereumNetworkConfig {
+     */
     network?: {
+        /**
+         * The network-wide identifier of this node. Should be unique
+         * within the Streamr Network.
+         */
         id?: string
+
+        /**
+         * Whether to accept proxy connections. Enabling this option allows
+         * the network node to act as proxy on behalf of other nodes / clients.
+         */
         acceptProxyConnections?: boolean
+
+        /**
+         * Defines the trackers that should be used for peer discovery and
+         * connection forming.
+         *
+         * Generally not intended to be configured by the end-user unless a
+         * custom network is being formed.
+         */
         trackers?: TrackerRegistryRecord[] | TrackerRegistryContract
+
+        /**
+         * Defines how often to ping connected tracker(s) to determine
+         * connection aliveness.
+         */
         trackerPingInterval?: number
+
+        /**
+         * Determines how often tracker connections should be maintained. This
+         * involves connecting to any relevant trackers to which a connection
+         * does not yet exist and disconnecting from irrelevant ones.
+         */
         trackerConnectionMaintenanceInterval?: number
+
+        /**
+         * When set to true private addresses will not be probed when forming
+         * WebRTC connections.
+         *
+         * Probing private addresses can trigger false-positive incidents in
+         * some port scanning detection systems employed by web hosting
+         * providers. Disallowing private addresses may prevent direct
+         * connections from being formed between nodes using IPv4 addresses
+         * on a local network.
+         *
+         * Details: https://github.com/streamr-dev/network/wiki/WebRTC-private-addresses
+         */
         webrtcDisallowPrivateAddresses?: boolean
+
+        /**
+         * Defines WebRTC connection establishment timeout.
+         *
+         * When attempting to form a new connection, if not established within
+         * this timeout, the attempt is considered as failed and further
+         * waiting for it will cease.
+         */
         newWebrtcConnectionTimeout?: number
+
+        /**
+         * Sets the low-water mark used by send buffers of WebRTC connections.
+         */
         webrtcDatachannelBufferThresholdLow?: number
+
+        /**
+         * Sets the high-water mark used by send buffers of WebRTC connections.
+         */
         webrtcDatachannelBufferThresholdHigh?: number
+
+        /**
+         * The maximum outgoing message size accepted by WebRTC connections.
+         * Messages exceeding the maximum size are simply discarded.
+         */
         webrtcMaxMessageSize?: number
+
+        /**
+         * Defines a custom UDP port range to be used for WebRTC connections.
+         * This port range should not be restricted by enclosing firewalls
+         * or VPC configurations.
```

Comment:
Virtual Port-Channel or Virtual private cloud? Probs best to add the full name here for searchability also.